libs/sdk-ui-dashboard/src/model/commandHandlers/layout/validation/itemValidation.ts

```diff
@@ -169,7 +169,7 @@ function* validateAndResolveKpiFilters(
 
         return {
             ...widget,
-            dateDataSet: measureDateDatasets[0]?.dataSet.ref,
+            dateDataSet: measureDateDatasets.dateDatasetsOrdered[0]?.dataSet.ref,
```

Comment:
is result of 

```
yield call(
            query,
            queryDateDatasetsForMeasure(widget.kpi.metric),
        );
```

correctly typed ? it is strange that previous impl lookup into different props

Reply:
according to typing, it should work, but I am not sure if the former typing is correct